missioncommand/emp3-web
# Feature/emp3 web#172 implement square editor

To test:
Create overlay, create square, edit feature. 

Ensure square maintains characteristics and only azimuth and height control points display and are removed.
